docs/book/helpers/doctype.md

```diff
@@ -45,15 +45,35 @@ And then print it out on top of your layout script:
 <?php echo $this->doctype() ?>
 ```
 
-Within an application based off the [laminas-mvc-skeleton](https://github.com/laminas/laminas-mvc-skeleton),
-you can specify the doctype via configuration:
+## Usage in a Mezzio Application
+
+The factory `Laminas\View\Helper\Service\DoctypeFactory` checks the application configuration, making it possible to
+define the doctype through your configuration, e.g. `config/autoload/mezzio.global.php`
+or a `ConfigProvider.php` in a module.
+
+For example, add the following lines to your `config/autoload/mezzio.global.php` file to set the `Doctype` to HTML5:
+
+```php
+return [
+    /* ... */
+    'view_helper_config' => [
+        'doctype' => \Laminas\View\Helper\Doctype::HTML5,
+    ],
+];
+```
+
+## Usage in a laminas-mvc Application
+
+If you're running a [laminas-mvc](https://docs.laminas.dev/laminas-mvc/) application, you should specify doctype via the
+[ViewManager](https://docs.laminas.dev/laminas-mvc/services/#viewmanager) service.
+
+Add the following lines to your `module/Application/config/module.config.php` file to set the `Doctype` to HTML5:
```

Comment:
```suggestion
Add the following lines to your `config/autoload/global.php` file to set the `Doctype` to HTML5:
```

This configuration is related to the entire application and not to a module. The same is already done in the previous section for Mezzio. (This is also wrong in the [skeleton application](https://github.com/laminas/laminas-mvc-skeleton/blob/12ff936b69caf63797b9c06a9f3a5d95482be913/module/Application/config/module.config.php#L48-L50).)

Reply:
Thanks! Yes, I was always wondering why global stuff was in the `module` config. Do you want me to submit a PR against laminas-mvc-skeleton to fix that?